src/components/NavCircle/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import "./style.css";
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import {faHome} from '@fortawesome/free-solid-svg-icons'
+import {faUser} from '@fortawesome/free-solid-svg-icons'
+import {faSignOutAlt} from '@fortawesome/free-solid-svg-icons'
+import {faPlus} from '@fortawesome/free-solid-svg-icons'
+
+function HandleClick (){
+document.getElementById('circularMenu').classList.toggle('active')
+}
+
+function NavCircle() {
+  return (
+    <div>
+      <div id="circularMenu" className="circular-menu">
```

Comment:
i'd rather have this as a state, but since there almost no requested changes, no need